refarch-apigateway/frontend/src/composables/formatter.ts

```diff
@@ -0,0 +1,15 @@
+import moment from "moment";
```

Comment:
We should remove moment as its deprecated. JavaScript API now natively supports the needed features.

Reply:
created issue #43